tests/qchem/test_tapering.py

```diff
@@ -952,3 +953,52 @@ def test_inconsistent_callable_ops(operation, op_wires, op_gen, message_match):
         taper_operation(
             operation, generators, paulixops, paulix_sector, wire_order, op_wires, op_gen
         )
+
+
+@pytest.mark.parametrize(
+    ("symbols", "geometry", "charge"),
+    [
+        (
+            ["H", "H"],
+            np.array([[0.0, 0.0, 0.0], [0.0, 0.0, 1.40104295]], requires_grad=True),
+            0,
+        ),
+        (
+            ["He", "H"],
```

Comment:
Not sure the rest of these molecules are needed. They increase the tests runtime for probably not a good reason. Having just H2 seems enough to me. You might try H2 with `max_size=2` and `max_size=5` instead.

Reply:
Updated!